.github/workflows/test.yml

```diff
@@ -61,33 +61,42 @@ jobs:
+          $sshPath = Join-Path $env:USERPROFILE ".ssh"
+          if (-not (Test-Path $sshPath)) {
+              New-Item -ItemType Directory -Path $sshPath -Force
+          }
           
+          $knownHostsPath = Join-Path $sshPath "known_hosts"
+          $githubKeys = (Invoke-RestMethod -Uri 'https://api.github.com/meta').ssh_keys
+          $githubKeys | ForEach-Object { "github.com $_" } | Out-File -FilePath $knownHostsPath -Append
+          
+          # Set permissions (600 in Windows terms)
+          $acl = Get-Acl $knownHostsPath
+          $acl.SetAccessRuleProtection($true, $false)
+          $rule = New-Object System.Security.AccessControl.FileSystemAccessRule($env:USERNAME, "Read, Write", "Allow")
+          $acl.AddAccessRule($rule)
+          Set-Acl $knownHostsPath $acl
+
       - name: Clone Packages To Use Locally
         run: |
             git clone git@github.com:decentraland/unity-explorer-packages.git            
-            jsonFilePath="Explorer/Packages/manifest.json"
-            sed -i 's|git@github.com:decentraland/unity-explorer-packages.git?path=/StylizedGrassShader|file:../../unity-explorer-packages/StylizedGrassShader|' "$jsonFilePath"
-            sed -i 's|git@github.com:decentraland/unity-explorer-packages.git?path=/StylizedWater2|file:../../unity-explorer-packages/StylizedWater2|' "$jsonFilePath"
-            sed -i 's|git@github.com:decentraland/unity-explorer-packages.git?path=/AVProVideo|file:../../unity-explorer-packages/AVProVideo|' "$jsonFilePath"
-            sed -i 's|git@github.com:decentraland/unity-explorer-packages.git?path=/SuperScrollView|file:../../unity-explorer-packages/SuperScrollView|' "$jsonFilePath"
-            sed -i 's|git@github.com:decentraland/unity-explorer-packages.git?path=/SoftMask|file:../../unity-explorer-packages/SoftMask|' "$jsonFilePath"
-            ls -l
-            # Display the content of manifest.json
-            cat "$jsonFilePath"
+            $jsonFilePath = "Explorer/Packages/manifest.json"
+            (Get-Content $jsonFilePath) `
+                -replace 'git@github.com:decentraland/unity-explorer-packages.git\?path=/StylizedGrassShader', 'file:../../unity-explorer-packages/StylizedGrassShader' `
+                -replace 'git@github.com:decentraland/unity-explorer-packages.git\?path=/StylizedWater2', 'file:../../unity-explorer-packages/StylizedWater2' `
+                -replace 'git@github.com:decentraland/unity-explorer-packages.git\?path=/AVProVideo', 'file:../../unity-explorer-packages/AVProVideo' `
+                -replace 'git@github.com:decentraland/unity-explorer-packages.git\?path=/SuperScrollView', 'file:../../unity-explorer-packages/SuperScrollView' `
+                -replace 'git@github.com:decentraland/unity-explorer-packages.git\?path=/SoftMask', 'file:../../unity-explorer-packages/SoftMask' |
+                Set-Content $jsonFilePath
+            
+            Get-ChildItem
+            Get-Content $jsonFilePath
```

Comment:
Why is this needed?

Reply:
We have issues with SSH, and this was workaround for that